src/Components/ReferenceTokenStatus.razor

```diff
@@ -0,0 +1,21 @@
+@using IdentityServerConfig.Models
+
+@if (!ReferenceTokenData.Status.Equals(ReferenceTokenDataModel.StatusCode.NotChecked))
+{
+    <div class="alert @(ReferenceTokenData.Status.Equals(ReferenceTokenDataModel.StatusCode.Active) ? "alert-success" : "alert-danger") " role="alert">
+        <h4 class="alert-heading">@ReferenceTokenData.Status.ToString()</h4>
+        @if (ReferenceTokenData.Status.Equals(ReferenceTokenDataModel.StatusCode.Invalid))
+        {
+            <p>The Reference Token for this client is invalid</p>
```

Comment:
Perhaps this should read `Reference token either does not exist or does not belong to this client `.